src/System.Reflection.TypeExtensions/tests/AssemblyExtensionTests.cs

```diff
@@ -26,7 +26,7 @@ public void GetModulesTest()
         public void GetTypes()
         {
             Assembly executingAssembly = GetType().GetTypeInfo().Assembly;
-            Assert.True(executingAssembly.GetTypes().Length >= 200);
+            Assert.True(executingAssembly.GetTypes().Length >= 175);
```

Comment:
We have fewer types (removed test classes and consolidated test data providers), so this has been updated